hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java

```diff
@@ -785,4 +789,40 @@ public void remove(String stateKey) {
     private static String extractStateKey(@NonNull final StateMetadata<?, ?> md) {
         return md.stateDefinition().stateKey();
     }
+
+    // FUTURE USE: the following code will become relevant with
+    // https://github.com/hashgraph/hedera-services/issues/11773
+    @NonNull
+    @Override
+    public SwirldState getSwirldState() {
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public PlatformState getPlatformState() {
+        throw new UnsupportedOperationException(
+                "To be implemented with https://github.com/hashgraph/hedera-services/issues/11773");
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void setPlatformState(@NonNull final PlatformState platformState) {
+        throw new UnsupportedOperationException(
+                "To be implemented with https://github.com/hashgraph/hedera-services/issues/11773");
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public String getInfoString(final int hashDepth) {
```

Comment:
I assume this isn't called very often. I'm not sure when it is called, but I cannot imagine what impact it would have on performance if it were!

Reply:
This method is called on relatively rare occasions:
- on start
- on reconnect 
- whenever we write hash file

So, its performance impact is minimal.